### PR TITLE
 Change the order of description of document's Docker rules in numerical order

### PR DESCRIPTION
**Pull request by zabio3**
### What I did

 -  Change the order of description of document's Docker rules in numerical order

#### Before
 - DL3005 -> DL3007 -> DL3006 -> DL3008

#### After
 - DL3005 -> DL3006 -> DL3007 -> DL3008

### CODE REVIEW

**Comment by lorenzo** [comment]
Thanks, would you mind fixing the markdown listing error that was reported?

**Comment by zabio3** [comment]
yes, please correct.

**Comment by zabio3** [comment]
Thanks for the comment! I will fix the problem areas.

File: src/Hadolint/Rules.hs [hunk 1]
```diff
@@ -447,8 +447,9 @@ aptGetPackages :: Shell.ParsedShell -> [String]
 aptGetPackages args =
     [ arg
     | cmd <- dropTarget <$> Shell.findCommands args
-    , arg <- Shell.getArgsNoFlags cmd
     , Shell.cmdHasArgs "apt-get" ["install"] cmd
+    , arg <- Shell.getArgsNoFlags cmd
+    , arg <- Shell.getArgsNoFlags cmd
```
Comment (lorenzo) [inline src/Hadolint/Rules.hs]: This line is duplicated

**Comment by lorenzo** [comment]
I'm sorry I did not notice the listing error was on the code and not on your change. There is no need for you to fix it now if you prefer. I can merge your original commit if you put your branch back to it

**Comment by lorenzo** [comment]
Thanks

**Pull request merged**